sys/key_value.js

```diff
@@ -191,14 +167,15 @@ class KVBucket {
                         (!req.headers['content-type'] ||
                         req.headers['content-type'] === oldContent.headers['content-type'])) {
                     hyper.metrics.increment(`sys_kv_${req.params.bucket}.unchanged_rev_render`);
-                    return {
+                    throw new HTTPError({
```

Comment:
Why are we throwing it now? Isn't returning the response enough?

Reply:
Should be enough, throwing is more explicit though.